client/src/modules/notification/model/actions.js

```diff
@@ -16,12 +20,21 @@ const removeNotification = payload => ({
 export const createNotificationWithTimeout = (
   dispatch,
   type,
-  notification,
+  message,
+  err = null,
   timeout = NOTIFICATION_TIMEOUT
 ) => {
+  const forbiddenError = err instanceof ForbiddenException;
+
   const id = uniqueId('notification_');
-  dispatch(addNotification({ id, type, notification }));
+  const delay = forbiddenError ? REDIRECT_TIMEOUT : timeout;
+  const notification = forbiddenError
+    ? { notificationId: 'user.auth.session-ended' }
```

Comment:
what if we gonna have other notifications with counter??
wouldn't be better to pass notifictaionID as parameter, as you do in regular notification via message parameter ??

Reply:
If we gonna have other notifications with I think we pass notificationId or some other prop from specific actions to the notification dispatcher... In `createNotificationWithTimeout` we will be able to decide what kind of notification should we display.
But in this case, in all actions in the app, I would have to check the type of the error and pass right notifactionId or dispatch different action. I didn't want to do all these things in all app's actions. So I decided to pass only error from catch to notification `createNotificationWithTimeout` and all the logic write in one place.